Assets/MixedRealityToolkit.SDK/Features/Input/Handlers/ControllerPoseSynchronizer.cs

```diff
@@ -32,8 +27,8 @@ public Handedness Handedness
         /// <inheritdoc />
         public bool DestroyOnSourceLost
```

Comment:
Why not just use 
```
public bool DestroyOnSourceLost => destroyOnSourceLost;
```
If you are not altering either the getter / setter?

Also consider this for other areas this is being changed for

Reply:
I believe that just creates a getter, no setter